File: app/src/main/java/com/fox/algorithm/数据结构与算法/leetcode/链表/_237删除链表中的节点.java
package com.fox.algorithm.数据结构与算法.leetcode.链表;

/**
 * @Author Fox
 * @Date 2021/3/27 10:34
 */
class _237删除链表中的节点 {

    public void deleteNode(ListNode node) {
        node.val = node.next.val;
        node.next = node.next.next;
    }
}

File: app/src/main/java/com/fox/algorithm/数据结构与算法/leetcode/链表/237删除链表中的节点.kt
package com.fox.algorithm.数据结构与算法.leetcode.链表

/**
 * @Author fox
 * @Date 2020/2/18 19:33
 */

fun deleteNode(node: ListNode) {
    node.`val` = node.next!!.`val`
    node.next = node.next!!.next
}
